src/assessment.js

```diff
@@ -19,7 +19,7 @@
 //
 //     `true` or `false`
 //
-var q1 = /* your answer here */;
+var q1 = false ;
```

Comment:
Incorrect. A variable inserted after line 7 would still be inside the scope of `deathStar`, and thus have access to `laserQuantity`.